src/main/java/com/powsybl/openloadflow/network/impl/LfBatteryImpl.java

```diff
@@ -98,6 +103,11 @@ public double getDroop() {
         return droop;
     }
 
+    @Override
+    public double getParticipationFactor() {
```

Comment:
I don't understand what will be the value returned if in the extension the participation is negative. We have a default value for the droop. Zero?

Reply:
yes, default value of (primitive) `double` is zero